index.js

```diff
@@ -105,6 +105,12 @@ module.exports = {
     // regular expressions.
     'unicorn/no-unsafe-regex': 'error',
 
+    // Nested ternaries are commonly used in JSX
+    // https://eslint.org/docs/rules/no-nested-ternary
+    'no-nested-ternary': 'off',
+    // https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/docs/rules/no-nested-ternary.md
+    'unicorn/no-nested-ternary': 'off',
```

Comment:
'unicorn/no-nested-ternary' is a bit different from 'no-nested-ternary'
>~Improved~ [Different] version of the no-nested-ternary ESLint rule, which allows cases where the nested ternary is only one level and wrapped in parens.

Might be good to explain why 'unicorn/no-nested-ternary' is disabled too? Presumably due to nesting level limitation? If that's true, also include why no limit on depth _might_ be okay. Be sure to address the "Nesting ternary expressions can make code more difficult to understand" claim of 'no-nested-ternary' and how this is mitigated.

Reply:
It's a redundant rule.